tests/monotouch-test/CoreGraphics/AffineTransformTest.cs

```diff
@@ -140,15 +140,27 @@ public void StaticMultiply ()
 		[Test]
 		public void Scale ()
 		{
-			var transform = CGAffineTransform.MakeTranslation (0, 200);
-			transform.Scale (1, -1);
-		
-			Assert.AreEqual ((nfloat) 1, transform.xx);
-			Assert.AreEqual ((nfloat) 0, transform.yx);
-			Assert.AreEqual ((nfloat) 0, transform.xy);
-			Assert.AreEqual ((nfloat) (-1), transform.yy);
-			Assert.AreEqual ((nfloat) 0, transform.x0);
-			Assert.AreEqual ((nfloat) (-200), transform.y0);
+			var transform1 = CGAffineTransform.MakeTranslation (1, 2);
+			// t' = t * [ sx 0 0 sy 0 0 ]
+			transform1.Scale (3, 4); // MatrixOrder.Prepend by default
```

Comment:
The new `Scale` code seems to use `Append` by default, right ?

Reply:
Right, wrong comment